src/storage/context.ts

```diff
@@ -1361,4 +1361,12 @@ export class StorageContext {
       isProofOverdue,
     }
   }
+
+  /**
+   * terminates the dataset by sending on-chain message
```

Comment:
```suggestion
   * Terminates the data set by sending on-chain message.
   * This will also result in the removal of all pieces in the data set.
```